auth/src/user.rs

```diff
@@ -244,9 +172,20 @@ impl UserManagement for UserManager {
         subscription_type: &models::user::SubscriptionType,
         subscription_quantity: i32,
     ) -> Result<(), Error> {
+        let mut transaction = self.pool.begin().await?;
+
+        if *subscription_type == models::user::SubscriptionType::Pro {
+            query("UPDATE users SET account_tier = $1 WHERE account_name = $2")
+                .bind(AccountTier::Pro.to_string())
+                .bind(name)
+                .execute(&mut *transaction)
+                .await?
+                .rows_affected();
```

Comment:
We don't need rows_affected here, right?

Reply:
Yes 🤦‍♂️ (copy-and-pasting 🙈 )